DOUBLE_CLICK_AND_EXECUTE_ME.py

```diff
@@ -15,65 +15,48 @@
 root.title("Dell'Osso Line Program")
 
 # Set window size
-#root.geometry("1920x1080")
+# root.geometry("800x600")
 root.attributes('-fullscreen', True)
 
 # Sets the background color
 root.configure(bg="black")
 
 # NEXT WINDOW heading
-label_heading = Label(root, text="PROCEED TO WINDOW", font=("Chalkduster", 100),
-                      fg="red", bg="black") 
-
-# Timer for NEXT WINDOW
-label_heading.timer = ' '
-
-# Pack label_heading
-label_heading.pack(pady=(30, 30))
-
-# Create a Frame to contain the numbers 1 through 6
-number_frame = Frame(root, bg="black")
-
-# Number 1 through 6
-nmbr_1 = Label(number_frame, text =" 1 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_1.pack(padx=15, side=LEFT)
-nmbr_2 = Label(number_frame, text =" 2 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_2.pack(padx=15, side=LEFT)
-nmbr_3 = Label(number_frame, text =" 3 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_3.pack(padx=15, side=LEFT)
-nmbr_4 = Label(number_frame, text =" 4 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_4.pack(padx=15, side=LEFT)
-nmbr_5 = Label(number_frame, text =" 5 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_5.pack(padx=15, side=LEFT)
-nmbr_6 = Label(number_frame, text =" 6 ", font=("SignPainter", 150), borderwidth=2,
-               relief="solid", bg="black", fg="black")
-nmbr_6.pack(padx=15, side=LEFT)
+row_heading_labels = []
+row_heading_labels.append(Label(root, text="PROCEED TO WINDOW", font=("Chalkduster", 100), fg="red", bg="black"))
+row_heading_labels[0].pack(pady=(30, 30))
+
+# Create number frames to hold the rows of numbers
+number_frames = []
+number_frames.append(Frame(root, bg="black"))
+number_frames.append(Frame(root, bg="black"))
```

Comment:
Ok so here you create an empty list then add two frames to the list. At first I was confused as to how you differentiate between the frames, but they have different index positions within the list that you can reference (as you do in line 37).

Reply:
Exactly.  All programmers start where you are starting.   They copy and paste code and adjust the parameters, but like some of the comments on reddit said.  It is best to try to use list and dictionaries and loops instead of copying and pasting the code and change the parameters.  It makes the codes a lot easier to maintain amd cleaner and usually easier to understand.

That being said I'm pretty impressed that this is your first project.  You had to learn a lot to pull of what you did. Nice work!!!